gnd/src/main/java/com/google/android/gnd/model/Terms.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.model;
+
+import com.google.auto.value.AutoValue;
+
+/** Represents a single application user. */
+@AutoValue
+public abstract class Terms {
```

Comment:
`TermsOfService` might be more self-describing.

Reply:
renamed to TermsOfService